src/sentry/tasks/statistical_detectors.py

```diff
@@ -50,6 +50,8 @@
 
 FUNCTIONS_PER_PROJECT = 100
 FUNCTIONS_PER_BATCH = 1_000
+TRANSACTIONS_PER_PROJECT = 50
```

Comment:
do you think we can try pushing for a 100?

Reply:
50 per project might be enough but I'm curious how far we can push this number

Reply:
👍 Sounds good. That 50 was extracted from existing code - can I punt increasing the number until https://github.com/getsentry/team-performance/issues/15? Seems like a good fit for that ticket since it's concerned with capacity planning anyway

Reply:
We should make this a runtime parameter that we can tune. whether we can push for of even 50 will be determined when we actually start turning up the traffic. At this point it's too soon to tell